### DISCUSSION TITLE
How to contribute to Appflowy

### DISCUSSION THREAD

**Discussion by Nike682631** [AppFlowy-IO/AppFlowy#670] [General help wanted]
Hi!!
I wanted to contribute to Appflowy and was wondering if the repo maintainers could assign me some beginner level issues I could work on.

**Comment by annieappflowy** [comment]
Hi @Nike682631 , welcome! I'm glad to hear that you are interested in contributing to AppFlowy. There are a few issues that are friendly to newcomers. You can find them here: https://github.com/orgs/AppFlowy-IO/projects/5/views/4

Please feel free to reach me out on Discord: https://discord.gg/9Q2xaN37tV
